infrastructure/images/terraform-azure-devops-agent/README.md

```diff
@@ -0,0 +1,4 @@
+# Altinn Terraform Azure DevOps Agent Image
+
+Image maintained by the platform team which installs some standard software that is used by our Terraform pipelines
+
```

Comment:
I think it would be nice if the README provided more context and some tips on how to maintain this image.
Particularly, what's the expected workflow for upgrading this image? When should those updates occur?
When/Should the [KUBECTL_VERSION](https://github.com/Altinn/altinn-platform/pull/1270/files#diff-4c5288c6beea9d21bf88000e0a8991773aca6cd9fc85e9e6cfc1768bdf8ca793R5) be updated?
Same with the [azure-devops-agent](https://github.com/Altinn/altinn-platform/pull/1270/files#diff-ef96150d14b36fc9e13ea6e5783eab5ece9bf2a40160b81ee13a4246ef61ee12R1) image version.
How should [.trivyignore](https://github.com/Altinn/altinn-platform/pull/1270/files#diff-99d06e1f6182b709352e48f09d7f12acd684ac9ec34180ced274b961a6204ae1R2) be handled?

Reply:
👍 added more context